XamlControlsGallery/ControlPages/RichEditBoxPage.xaml.cs

```diff
@@ -197,6 +200,12 @@ private void REBCustom_Unloaded(object sender, RoutedEventArgs e)
                 REBCustom.SelectionFlyout.Opening -= Menu_Opening;
                 REBCustom.ContextFlyout.Opening -= Menu_Opening;
             }
-        } 
+        }
+
+        private void Editor_TextChanged(object sender, RoutedEventArgs e)
+        {
+            editor.Document.Selection.CharacterFormat.ForegroundColor = currentColor;
+
```

Comment:
Nit: please remove the extra line of whitespace

Reply:
Fixed with latest commit